todo_app/get_items_trello.py

```diff
@@ -0,0 +1,33 @@
+import requests
```

Comment:
These files are really useful for testing, and definitely worth writing, but I'm a fan of then removing them once the core codebase is working with that functionality built in - they won't be maintained so they're not a good reference long term